oracle-plugin/src/e2e-test/java/io.cdap.plugin/OracleClient.java

```diff
@@ -52,6 +62,108 @@ public static int countRecord(String table, String schema) throws SQLException,
+            byte[] sourceArrayBfile = rsSource.getBytes(count);
+            byte[] targetArrayBfile = rsTarget.getBytes(count);
+            if (!Arrays.equals(sourceArrayBfile, targetArrayBfile)) {
+              Assert.assertEquals(String.format("Different values found for column : %s", columnName),
+                                  sourceArrayBfile, targetArrayBfile);
+              differentValues = true;
+            }
+            break;
+          case Types.BLOB:
+            Blob blobSource = rsSource.getBlob(count);
+            byte[] sourceArrayBlob = blobSource.getBytes(1, (int) blobSource.length());
+            Blob blobTarget = rsTarget.getBlob(count);
+            byte[] targetArrayBlob = blobTarget.getBytes(1, (int) blobTarget.length());
+            if (!Arrays.equals(sourceArrayBlob, targetArrayBlob)) {
+              Assert.assertEquals(String.format("Different values found for column : %s", columnName),
+                                  sourceArrayBlob, targetArrayBlob);
+              differentValues = true;
+            }
+            break;
+          case Types.CLOB:
+            Clob clobSource = rsSource.getClob(count);
+            String sourceClobString = clobSource.getSubString(1, (int) clobSource.length());
+            Clob clobTarget = rsTarget.getClob(count);
+            String targetClobString = clobTarget.getSubString(1, (int) clobTarget.length());
+            if (!sourceClobString.equals(targetClobString)) {
+              Assert.assertEquals(String.format("Different values found for column : %s", columnName),
+                                  sourceClobString, targetClobString);
+              differentValues = true;
+            }
+            break;
+          case Types.TIMESTAMP:
+            GregorianCalendar gc = new GregorianCalendar(TimeZone.getTimeZone("UTC"));
+            gc.setGregorianChange(new Date(Long.MIN_VALUE));
+            Timestamp sourceTS = rsSource.getTimestamp(count, gc);
+            Timestamp targetTS = rsTarget.getTimestamp(count, gc);
+            if (!sourceTS.equals(targetTS)) {
+              Assert.assertEquals(String.format("Different values found for column : %s", columnName),
+                                  rsSource.getTimestamp(count, gc), rsTarget.getTimestamp(count, gc));
+              differentValues = true;
+            }
+            break;
+          default:
+            String sourceString = rsSource.getString(count);
+            String targetString = rsTarget.getString(count);
+            if (!String.valueOf(sourceString).equals(String.valueOf(targetString))) {
+              Assert.assertEquals(String.format("Different values found for column : %s", columnName),
+                                  sourceString, targetString);
+              differentValues = true;
+            }
+        }
+        count++;
+      }
+    }
+    Assert.assertTrue("Number of rows in Source table is greater than the number of rows in Target table",
```

Comment:
This does not looks right please check the condition.

Reply:
Corrected the condition